main.go

```diff
@@ -231,10 +235,43 @@ func paConnectionWatchdog(ctx *ntcontext) {
 		ctx.paClient = paClient
 		go updateNoiseSupressorLoaded(paClient, &ctx.noiseSupressorState)
 
-		ctx.inputList = getSources(paClient)
+		ctx.inputList = getSourcesWithPreSelectedInput(paClient, ctx.config)
```

Comment:
paClient and config are both accessible from `ctx`, I'd prefer to pass ctx here.

Reply:
Makes sense, pushed the change.